Update screenshots and style. (#14656)
cockroachdb/docs: v22.1/demo-fault-tolerance-and-recovery.md

    {% include_cached copy-clipboard.html %}
    ~~~ shell
    $ cockroach init --insecure --host=localhost:26257
    ~~~

## Step 2. Set up load balancing

> In this module, you run a sample workload to simulate multiple client connections. Each node is an equally suitable SQL gateway for the load, but it's always recommended to [spread requests evenly across nodes](recommended-production-settings.html#load-balancing). This section shows how to set up the open-source [HAProxy](http://www.haproxy.org/) load balancer.

Reviewer: ```suggestion
In this tutorial, you will run a sample workload to simulate multiple client connections. Each node is an equally suitable SQL gateway for the load, but it's always recommended to [spread requests evenly across nodes](recommended-production-settings.html#load-balancing). This section shows how to set up the open-source [HAProxy](http://www.haproxy.org/) load balancer.
```

1. Go to the DB Console at <a href="http://localhost:8080" data-proofer-ignore>http://localhost:8080</a>.

2. To check the SQL queries getting executed, click **Metrics** on the left, and hover over the **SQL Statements** graph at the top:

    <img src="{{ 'images/v22.1/fault-tolerance-1.png' | relative_url }}" alt="DB Console SQL Statements" style="border:1px solid #eee;max-width:100%" />

3. To check the client connections from the load generator, select the **SQL** dashboard and hover over the **Open SQL Sessions** graph:

    <img src="{{ 'images/v22.1/fault-tolerance-2.png' | relative_url }}" alt="DB Console Open Sessions" style="border:1px solid #eee;max-width:100%" />

    You'll notice 3 client connections from the load generator. If you want to check that HAProxy balanced each connection to a different node, you can change the **Graph** dropdown from **Cluster** to specific nodes.

> 4. To see more details about the `ycsb` database and `usertable` table, click **Databases** in the upper left and click **ycsb**:

Reviewer: Based on the new screenshots, does the table need to be called `public.username` throughout this doc? Not sure of the technicalities.

    503 53651     1   0 10:18AM ttys013    0:38.81 cockroach start --insecure --store=fault-node2 --listen-addr=localhost:26258 --http-addr=localhost:8081 --join=localhost:26257,localhost:26258,localhost:26259
    503 53671     1   0 10:18AM ttys013    0:35.40 cockroach start --insecure --store=fault-node3 --listen-addr=localhost:26259 --http-addr=localhost:8082 --join=localhost:26257,localhost:26258,localhost:26259
    503 53684     1   0 10:18AM ttys013    0:38.44 cockroach start --insecure --store=fault-node4 --listen-addr=localhost:26260 --http-addr=localhost:8083 --join=localhost:26257,localhost:26258,localhost:26259
    503 53708     1   0 10:18AM ttys013    0:39.49 cockroach start --insecure --store=fault-node5 --listen-addr=localhost:26261 --http-addr=localhost:8084 --join=localhost:26257,localhost:26258,localhost:26259
    503 53730     1   0 10:18AM ttys013    0:33.37 cockroach start --insecure --store=fault-node6 --listen-addr=localhost:26262 --http-addr=localhost:8085 --join=localhost:26257,localhost:26258,localhost:26259
    ~~~

> 1. Gracefully shut down the node stored in `fault-node5`, specifying its process ID:

Reviewer: It now seems strange that we only cite a specific process ID here. Perhaps this can clarify:

```suggestion
1. Gracefully shut down the node stored in `fault-node5`, specifying its process ID (in this example, `53708`):
```

    --background
    ~~~

1. Use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) command to change the cluster's `default` replication factor to 5:

    {% include_cached copy-clipboard.html %}
    ~~~ shell
    $ cockroach sql --insecure --host=localhost:26000 \
    --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=5;"
    ~~~

1. In the DB Console **Overview** dashboard, watch the replica count increase and even out across all 6 nodes:

    <img src="{{ 'images/v22.1/fault-tolerance-8.png' | relative_url }}" alt="DB Console Cluster Restore" style="border:1px solid #eee;max-width:100%" />

    This shows the cluster up-replicating so that each range has 5 replicas, one on each node.

## Step 9. Simulate two simultaneous node failures

Gracefully shut down **2 nodes**, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):

{% include_cached copy-clipboard.html %}
~~~ shell

> kill -TERM {process-id}

Reviewer: This change makes it look as though only one pid is referenced. We should specify exact process IDs from the example, for clarity:

```
{% include_cached copy-clipboard.html %}
~~~ shell
kill -TERM 53708 53730
~~~
```

Author: I can't specify `53708` here because that process no longer exists since it was killed in a preceding step. That's why I give the generic kill example and say to shut down 2 nodes.  When a user runs this tutorial the process IDs will be different anyway.

Reviewer: That node is restarted in Step 8. I assumed that it would have the same process ID, but I can't recall what happened when I tested these steps. 

But your latter point is taken. In that case, I think the command should read

```
{% include_cached copy-clipboard.html %}
~~~ shell
kill -TERM {process IDs}
~~~
```

Author: It doesn't have the same process ID. I think this solution is much better. Thanks!

~~~

## Step 10. Check cluster status and service continuity

1. Click **Overview** on the left, and verify the state of the cluster:

    <img src="{{ 'images/v22.1/fault-tolerance-9.png' | relative_url }}" alt="DB Console Cluster Health" style="border:1px solid #eee;max-width:100%" />

1. To verify that the cluster still serves data, use the `cockroach sql` command to interact with the cluster.

>     Count the number of rows in the `ycsb.usertable` table to see that it serves reads:

Reviewer: Again, not sure if this needs to be something like `ycsb.public.usertable`. I'm still naive to how the table names are displayed, so apologies if this is irrelevant!

    --background
    ~~~

1. Use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) command to change the cluster's `default` replication factor to 5:

    {% include_cached copy-clipboard.html %}
    ~~~ shell
    $ cockroach sql --insecure --host=localhost:26000 \
    --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=5;"
    ~~~

1. In the DB Console **Overview** dashboard, watch the replica count increase and even out across all 6 nodes:

    <img src="{{ 'images/v22.1/fault-tolerance-8.png' | relative_url }}" alt="DB Console Cluster Restore" style="border:1px solid #eee;max-width:100%" />

    This shows the cluster up-replicating so that each range has 5 replicas, one on each node.

## Step 9. Simulate two simultaneous node failures

Gracefully shut down **2 nodes**, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):

{% include_cached copy-clipboard.html %}
~~~ shell

> kill -TERM {process-id}

Author: ```suggestion
kill -TERM {process IDs}
```

    $ pkill haproxy
    ~~~

3. Gracefully shut down the remaining **4 nodes**, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure) and the new process ID for the node stored in `fault-node5`:

    {% include_cached copy-clipboard.html %}
    ~~~ shell

>     kill -TERM {process-id}

Reviewer: ```suggestion
    kill -TERM 53645 53651 53671 53684
```

Author: Same comment as above. The instructions say to shut down the remaining 4 nodes and I give the command for that here. The users\'s process IDs will be different.

Reviewer: Same suggestion as above:

{% include_cached copy-clipboard.html %}
~~~ shell
kill -TERM {process IDs}
~~~

>     kill -TERM {process-id}

Author: ```suggestion
    kill -TERM {process IDs}
```
